cost-analyzer/values.yaml

```diff
@@ -361,10 +361,8 @@ kubecostMetrics:
     serviceMonitor: # the kubecost included prometheus uses scrapeConfigs and does not support service monitors. The following options assume an existing prometheus that supports serviceMonitors.
       enabled: false
       additionalLabels: {}
-      networkCosts:
```

Comment:
Did not see any references to `.Values.kubecostMetrics.exporter.serviceMonitor.networkCosts`

Reply:
@antoinedeschenes Thanks for the catch, I don't see references to that config either!

@jessegoodier Do you happen to know if this config `.Values.kubecostMetrics.exporter.serviceMonitor.networkCosts` is currently or previously in use?

Reply:
Not to my knowledge, and a quick search seems to validate this hasn't been added.

I'm curious- what would a common relabeling be used for?

Reply:
A few examples:

I'm removing these labels added pre-scrape: `namespace|container|endpoint|service`, as I don't want to waste
Prometheus memory storing constant labels. For `pod` and `instance`, I drop/hardcode them to prevent series from being duplicated if the cost-analyzer pod restarts.

```yaml
    relabelings:
      - action: labeldrop
        regex: namespace|pod|container|endpoint|service
      - targetLabel: instance
        sourceLabels: [__meta_kubernetes_service_name]
```

As for metricRelabelings (post-scrape), I can use them to drop the `go_.*` performance metrics, or in this case, the gpu-related series, since disabledMetrics has some issues (https://github.com/opencost/opencost/pull/1897)

```yaml
    metricRelabelings:
      - action: drop
        regex: (go_|.*_gpu).*
        sourceLabels:
          - __name__
```

Reply:
Awesome, thanks for sharing @antoinedeschenes!

I'm currently unsure whether dropping those labels have an impact on cost-analyzer's calculations. I'll need to test this myself and investigate more in the future!